CONTRIBUTING.md

```diff
@@ -126,6 +126,11 @@ Examples:
   - For a new translation of an existing page: `cp: add Tamil translation`
   - For related changes to several pages: `grep, find, locate: synchronize format of wildcards`
 
+## Inclusive language
+
+Where possible, use inclusive language in the content of pages. For example, prefer terms like "denylist"/"allowlist" instead of "blacklist"/"whitelist", "primary"/"secondary" instead of "master"/"slave", "one" instead of "him"/"her", etc.
+Of course, this shouldn't sacrifice content clarity, such as when documenting tools where such terms have specific technical meanings, and their usage is central to explaining the involved concepts.
```

Comment:
Sorry, I just now realized that my suggested sentence had two "such" in close proximity. How about we swap "such as" for "e.g." (or the spelled-out "for example", if you prefer)?

Reply:
Swapped for "the terminology"